components/admin_console/custom_plugin_settings/index.ts

```diff
@@ -71,6 +71,7 @@ function makeGetPluginSchema() {
                     return {
                         ...setting,
                         type,
+                        fromPlugin: true,
```

Comment:
This is needed to tell the `SchemaAdminSettings` component that this is a plugin component it is rendering